website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -1156,26 +1184,30 @@ List of extensions to modify Envoy proxy configuration. Refer to [Envoy Extensio
 
 #### Values
 
-You can configure the following parameters in the `EnvoyExtensions` block:
+The following table describes how to configure values in the `envoyExtensions` map:
 
 | Parameter | Description | Data type | Default |
-| ---   | ---         | ---       | ---     |
-| `name` | Name of the extension. | string | `""` |
-| `required` | When Required is true and the extension does not update any Envoy resources, an error is returned. Use this parameter to ensure that extensions required for secure communication are not unintentionally bypassed. | string | `""` |
-| `arguments` | Arguments to pass to the extension executable. | map | `nil` |
+| ---       | ---         | ---       | ---     |
+| `name` | Specifies the name of the extension. | String | None |
+| `required` | Specify `true` to require Consul to implement the extension. <p>Use this parameter to ensure that extensions required for secure communication are not unintentionally bypassed.</p><p>Consul returns an error when `required` is set to `true` and the extension does not update Envoy resources. </p> | String | None |
+| `arguments` | Specifies the arguments to pass to the extension executable. Refer to the documentation for the extension you want to implement for additional information. | Map | None |
+| `consulVersion` | Specifies the version of Consul that the extension is allowed to work with. Consul validates the version during xDS updates. If a different version is in use, Consul skips the extension and writes the event to the log. <p>The `consulVersion` and `envoyVersion` must both validate for Consul to implement the extension.</p> | String | None |
+| `envoyVersion` | Specifies the version of Envoy that the extension is allowed to work with. Consul validates the version during xDS updates. If a different version is in use, Consul skips the extension and writes the event to the log. <p>The `consulVersion` and `envoyVersion` must both validate for Consul to implement the extension.</p> | String | None |
```

Comment:
```suggestion
| `consulVersion` | Specifies the Consul [version constraint](https://github.com/hashicorp/go-version) for the extension. Consul validates this against its runtime version during xDS updates. If a non-matching version is in use, Consul will log and skip the extension. <p>Use this parameter to avoid upgrade issues when a configured extension is not compatible with a new version of Consul.</p> | String | None |
| `envoyVersion` | Specifies the Envoy [version constraint](https://github.com/hashicorp/go-version) for the extension. Consul validates this against the version of the running Envoy proxy during xDS updates. If a non-matching version is in use, Consul will log and skip the extension. <p>Use this parameter to avoid upgrade issues when a configured extension is not compatible with a new version of Envoy.</p> | String | None |
```